app/src/main/res/layout/item_contact.xml

```diff
@@ -39,15 +39,17 @@
         android:layout_toRightOf="@+id/item_contact_icon"
         android:layout_toStartOf="@+id/item_contact_action_message"
         android:ellipsize="end"
-        android:singleLine="true"
-        android:paddingRight="2px"
-        android:text="@string/stub_contact"
-        android:textColor="?attr/resultColor"
+        android:paddingEnd="2dp"
```

Comment:
should we extend the TextView to include shadow with the size in dp?
Example: https://stackoverflow.com/a/12278391/6761564

Reply:
Not sure that's worth it honestly